scripts/patchFiles.ts

```diff
@@ -43,7 +43,7 @@ async function main() {
   const bugsnagFiles = [
     './android/app/src/main/AndroidManifest.xml',
     './ios/edge/Info.plist',
-  ];
+  ].filter(file => fs.existsSync(file));
```

Comment:
Why would these files not exist?

Reply:
Well, Airbitz doesn't use Bugsnag.

Reply:
Shouldn't we just remove this code then?

Reply:
We could, but I want to cherry-pick this commit onto edge-react-gui.